src/AWS/Signers/V4.elm

```diff
@@ -44,6 +44,7 @@ algorithm =
 headers : AWS.Config.Service -> List ( String, String )
 headers config =
     [ ( "Host", AWS.Http.host config.endpoint )
+    , ( "Accept", "application/json" )
```

Comment:
Should all service calls set this as a header? Or is this just for testing purposes?